### PR TITLE
Adding clarifications to avoid common issues our users currently encounter 

### PR DESCRIPTION
**Pull request by marcoam**
Updating K4K8s Enterprise instructions to add warning about deleting Postgres PV and enabling admin's API

### CODE REVIEW

File: app/enterprise/1.5.x/kong-for-kubernetes/install-on-kubernetes.md [hunk 1]
```diff
@@ -237,6 +241,9 @@ The steps in this section show you how to install Kong Enterprise on Kubernetes
     $ kubectl get svc -n kong my-kong-kong-admin --output=jsonpath='{.status.loadBalancer.ingress[0].ip}'
     ```
 
+⚠️**Important:** The command above requires the enabling of the Admin API. If you have not set the value `admin.enabled` to true in your `values.yaml`, then the command above will not work.
```
Comment (lena-larionova) [inline app/enterprise/1.5.x/kong-for-kubernetes/install-on-kubernetes.md:244]:
```suggestion
    <div class="alert alert-warning">
    <i class="fas fa-exclamation-triangle" style="color:orange; margin-right:3px"></i> 
    <strong>Important:</strong> The command above requires the Kong Admin API. If you 
    have not set <code>admin.enabled</code> to <code>true</code> in your 
    <code>values.yaml</code>, then this command will not work. 
    </div>
```

**Comment by marcoam** [comment]
> Can you also make these changes in 2.1.x?

done @lena-larionova

**Comment by hbagdi** [comment]
lgtm, thanks for the change marco

File: app/enterprise/1.5.x/kong-for-kubernetes/install-on-kubernetes.md [hunk 2]
```diff
@@ -216,6 +216,12 @@ The steps in this section show you how to install Kong Enterprise on Kubernetes
     the database, delete Postgres' persistent volumes in your Kubernetes cluster, then 
     retry the Helm install. 
     </div>
+    
+    <div class="alert alert-warning">
+    <i class="fas fa-exclamation-triangle" style="color:orange; margin-right:3px"></i> 
+    <strong>Important:</strong> 
+    If you have already installed the CRDs, run the command above with the following flag: `--set ingressController.installCRDs=false`.
```
Comment (lena-larionova) [inline app/enterprise/1.5.x/kong-for-kubernetes/install-on-kubernetes.md:223]:
```suggestion
    If you have already installed the CRDs, run the command above with the following flag: <code>--set ingressController.installCRDs=false</code>. 
```

File: app/enterprise/2.1.x/kong-for-kubernetes/install-on-kubernetes.md [hunk 3]
```diff
@@ -223,6 +223,12 @@ The steps in this section show you how to install Kong Enterprise on Kubernetes
     retry the Helm install. 
     </div>
 
+    <div class="alert alert-warning">
+    <i class="fas fa-exclamation-triangle" style="color:orange; margin-right:3px"></i> 
+    <strong>Important:</strong> 
+    If you have already installed the CRDs, run the command above with the following flag: `--set ingressController.installCRDs=false`.
```
Comment (lena-larionova) [inline app/enterprise/2.1.x/kong-for-kubernetes/install-on-kubernetes.md:229]:
```suggestion
    If you have already installed the CRDs, run the command above with the following flag: <code>--set ingressController.installCRDs=false</code>. 
```

**Comment by marcoam** [comment]
Hi @lena-larionova - looks good, thanks for the feedback!

**Pull request merged**